qbft/spectest/tests/controller/futuremsg/test.go

```diff
@@ -53,7 +55,10 @@ func (test *ControllerSyncSpecTest) Run(t *testing.T) {
 
 	r, err := contr.GetRoot()
 	require.NoError(t, err)
-	require.EqualValues(t, test.ControllerPostRoot, hex.EncodeToString(r[:]))
+	if test.ControllerPostRoot != hex.EncodeToString(r[:]) {
```

Comment:
```suggestion
	require.EqualValues(t, test.ControllerPostRoot, hex.EncodeToString(r[:]), "Post state not equal %v", typescomparable.PrintDiff(contr, test.ControllerPostState))
```

Reply:
This will have to change in more than one place.

If we change it in more than place, we might as well do it correctly and print a better diff...
But this is not for this PR